app/services/unpublish_handler_service.rb

```diff
@@ -9,18 +9,17 @@ def call(content_id, redirect)
     case type
     when :taxon_tree
       unsubscribe(subscriber_lists, redirect, taxon_template)
+    when :policy_areas
+      unsubscribe(subscriber_lists, redirect, policy_area_template)
+    when :policies
+      unsubscribe(subscriber_lists, redirect, policy_template)
     else
-      unsubscribe_other(subscriber_lists)
+      log_non_taxon_lists(subscriber_lists)
```

Comment:
This method name looks like it needs tweaking now.

Reply:
fixed